modelscope/preprocessors/nlp/codegeex_preprocessor.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2022 Zhipu.AI
+
+import re
+from typing import Any, Dict, Iterable, Optional, Tuple, Union
+
+from modelscope.metainfo import Models, Preprocessors
+from modelscope.preprocessors.base import Preprocessor
+from modelscope.preprocessors.builder import PREPROCESSORS
+from modelscope.utils.constant import Fields, InputFields, ModeKeys, ModelFile
+from modelscope.utils.type_assert import type_assert
+
+
+@PREPROCESSORS.register_module(Fields.nlp, module_name=Preprocessors.codegeex)
+class CodeGeeXPreprocessor(Preprocessor):
+
```

Comment:
if no preprocess is needed, you can overload  your preprocess function in your pipeline as below without define a preprocessor class

```python
    def preprocess(self, inputs: Input, **preprocess_params) -> Dict[str, Any]:
           return inputs
```